### PR TITLE
Dzollo/smoothpose init v2.1 master

### PR DESCRIPTION
**Pull request by denniszollo**
Changes to support piksi_ins required for factory provisioning. 

Also make smoothpose run without root permissions.

### CODE REVIEW

File: package/piksi_ins/overlay/etc/init.d/S96piksi_ins [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+name="PoseDaemon"
+cmd="PoseDaemon"
+dir="/"
+user="piksi_ins"
+
+PERSIST_DIR=/persistent/
```
Comment (silverjam) [inline package/piksi_ins/overlay/etc/init.d/S96piksi_ins]: Doesn't look used, remove?

**Comment by denniszollo** [comment]
We also cannot merge this until we change the Smoothpose wrapper permissions;

it currently writes to /etc/ and /root

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.0.0-develop-2018091419-13-g9431a2f9
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=680452)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-13-g9431a2f9&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-13-g9431a2f9&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.0.0-develop-2018091419-13-g9431a2f9
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.0.0-develop-2018091419-13-g9431a2f9)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.0.0-develop-2018091419-13-g9431a2f9/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.0.0-develop-2018091419-13-g9431a2f9

**Comment by silverjam** [comment]
Any idea what's going on here?

```
>>>   Copying overlay /piksi_buildroot/package/orion_daemon/overlay
>>>   Copying overlay git@github.com:carnegieroboticsllc/piksi_ins.git/overlay
Failed to add the RSA host key for IP address '192.30.253.113' to the list of known hosts (/home/travis/.ssh/known_hosts).
Invalid command: 'rsync --server --sender -logDtprIe.iLsfxC . carnegieroboticsllc/piksi_ins.git/overlay/'
  You appear to be using ssh to clone a git:// URL.
  Make sure your core.gitProxy config option and the
  GIT_PROXY_COMMAND environment variable are NOT set.
rsync: connection unexpectedly closed (0 bytes received so far) [Receiver]
rsync error: error in rsync protocol data stream (code 12) at io.c(235) [Receiver=3.1.2]
make[3]: *** [target-finalize] Error 12
Makefile:697: recipe for target 'target-finalize' failed
Makefile:79: recipe for target '_all' failed
make[2]: *** [_all] Error 2
make[2]: Leaving directory '/piksi_buildroot/buildroot'
Makefile:63: recipe for target 'image-release-protected' failed
make[1]: Leaving directory '/piksi_buildroot'
make[1]: *** [image-release-protected] Error 2
Makefile:67: recipe for target 'image-release-ins' failed
make: *** [image-release-ins] Error 2
make: *** [docker-make-image-release-ins] Error 2
```

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.0.0-develop-2018091419-14-g5962c62e
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=680790)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-14-g5962c62e&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-14-g5962c62e&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.0.0-develop-2018091419-14-g5962c62e
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.0.0-develop-2018091419-14-g5962c62e)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.0.0-develop-2018091419-14-g5962c62e/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.0.0-develop-2018091419-14-g5962c62e

**Comment by anonymous-travis-swiftnav** [comment]
## HITL smoke tests: v2.0.0-develop-2018091419-16-g8f04c76f
These test runs are kicked off whenever you push a new commit to this PR. All passfail metrics in these runs must pass for the `hitl/pass-fail` status to be marked successful.
### job status
+ [live-roof-650-townsend-post runs](https://gnss-analysis.swiftnav.com/jobs/id=683760)
### gnss-analysis results
At least one run must complete for these links to have data.
#### passfail
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=pass_fail&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-16-g8f04c76f&groupby_key=firmware&display_type=table)
#### detailed
+ [live-roof-650-townsend-post](https://gnss-analysis.swiftnav.com/summary_type=q50&metrics_preset=detailed&scenario=live-roof-650-townsend-post&build_type=buildroot_pr&firmware_versions=v2.0.0-develop-2018091419-16-g8f04c76f&groupby_key=firmware&display_type=table)

**Comment by anonymous-travis-swiftnav** [comment]
## v2.0.0-develop-2018091419-16-g8f04c76f
+ [Artifacts (HITL Dashboard Page)](https://gnss-analysis.swiftnav.com/artifacts/build_type=buildroot_pr&name_filter=v2.0.0-develop-2018091419-16-g8f04c76f)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v2.0.0-develop-2018091419-16-g8f04c76f/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v2.0.0-develop-2018091419-16-g8f04c76f

**Pull request merged**